kimchi/src/proof.rs

```diff
@@ -181,6 +184,47 @@ impl<G: AffineCurve> RecursionChallenge<G> {
     pub fn new(chals: Vec<G::ScalarField>, comm: PolyComm<G>) -> RecursionChallenge<G> {
         RecursionChallenge { chals, comm }
     }
+
+    pub fn evals(
```

Comment:
maybe a more descriptive name and some doc comment before I review this function :D ?

Reply:
I'll still point out that if you're moving that function around, it's not a bad idea to add a doc comment ;) I'm actually trying to understand what it does, and even though I already read/understood that function it takes a while every time without docs :o